Symfony/CS/Tests/Config/ConfigTest.php

```diff
@@ -23,7 +23,7 @@ public function testThatDefaultFinderWorksWithDirSetOnConfig()
         $config->setDir(__DIR__.'/../Fixtures/FinderDirectory');
 
         $iterator = $config->getFinder()->getIterator();
-        $this->assertSame(1, count($iterator));
+        $this->assertCount(1, $iterator);
```

Comment:
took some digging, but here you go :)
https://github.com/sebastianbergmann/phpunit/pull/1413